diagnostic.md

```diff
@@ -34,20 +34,15 @@ Open up this file in Atom, and write your answers below (where indicated).
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+Relative b/c desktop is relative to blah and the users directory. it's like saying the bank is next to the post office.
```

Comment:
Absolute. When a path starts with it's root `/Users/` you know it's an absolute path.